Makefile

```diff
@@ -1,4 +1,5 @@
 test_build_type?=debug
+ca_cert_file?=avito_ca.crt
```

Comment:
Do you propose to store it in the project?
What about `~/.kube`?

Reply:
Fixed